.scripts/build_steps.sh

```diff
@@ -32,9 +32,9 @@ pkgs_dirs:
 CONDARC
 
 
-mamba install --update-specs --yes --quiet --channel conda-forge \
+conda install --update-specs --yes --quiet --channel conda-forge \
     conda-build pip boa conda-forge-ci-setup=3
-mamba update --update-specs --yes --quiet --channel conda-forge \
+conda install --update-specs --yes --quiet --channel conda-forge \
```

Comment:
This is temporary and should be reverted once https://github.com/conda-forge/docker-images/pull/236 is released.

Reply:
@johnhany97 
Just to be sure, was it on purpose to change this to `conda install` instead of `conda update` ?

Reply:
I changed to `conda install` because `conda update` failed and asked me to run `install` instead.